target/linux/ramips/image/Makefile

```diff
@@ -159,6 +170,8 @@ endef
 
 define Device/Default
   PROFILES = Default
+  KERNEL_LOADADDR := $(loadaddr-y)
+  LZMA_TEXT_START := $(lzmatextstart-y)
```

Comment:
If I've checked correctly, you don't need lzmatextstart-y here at all, but could just put the literal value here.

Reply:
Yes, we can just put the literal value here without using lzmatextstart-y. I've removed lzmatextstart-y. Done.

I have made all the changes that you have requested. FW test on a real HW was successful. Many thanks for the review!